platform-api/src/routers/forms.py

```
@@ -0,0 +1,125 @@
+
+    # Fire and forget to Discord
+    await notify_discord(body, webhook_url)
+
+    return SubmissionResponse(id=doc_id)
+
+@router.get("/submissions", response_model=List[SubmissionListItem])
+async def list_submissions(
+    site: str = Query("capstone", description="Target site/workspace"),
+    status: str = Query("submitted", description="Submission status filter (use 'all' for no filter)"),
+    limit: int = Query(20, ge=1, le=100, description="Max records to return"),
+    sanity: SanityClient = Depends(get_sanity),
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)